### DISCUSSION TITLE
Adding grid template in JIT mode

### DISCUSSION THREAD

**Discussion by hamza512b** [tailwindlabs/tailwindcss#5866] [Help] [answered]
Hi!
What if we could something like this:
```html
<div class="grid grid-template-columns-[8fr,1fr,1fr]">
    <div></div>
    <div></div>
    <div></div>
</div>
```

Where `grid-template-columns-[8fr,1fr,1fr]` correspond:
```
grid-template-columns: 8fr 1fr 1fr;
```

Thank you

**Comment by dance2die** [comment, marked as answer]
You can use `grid-cols-[8fr,1fr,1fr]` to generate dynamic grid template columns.
**Demo**: https://play.tailwindcss.com/NaGAbsaUmr

![image](https://user-images.githubusercontent.com/8465237/138620435-0d446927-b3a5-4128-9a03-e8044fde4081.png)

**Reply by hamza512b** [reply]
Thank you, I didn't know that.
